### PR TITLE
Replace use of Powershell scripts using SqlClient and Process.Start

### PR DESCRIPTION
**Pull request by Antonstockmarr**
CommandExecutor can now execute commands without PowerShell using `Process.Start`.
`Invoke-SqlCmd` has been removed. Instead `SqlCommand` is used to query the database.

Resolves remaining items on #10

### CODE REVIEW

**Review by AndreasHassing — CHANGES_REQUESTED** [review]
RFC, looks good other than a few nits 😊.

File: src/ScaleUnitManagement/ScaleUnitFeatureManager/Common/RunDBSync.cs [hunk 1]
```diff
@@ -27,23 +27,23 @@ public void Run()
...
-                    dbSyncTool
-                    + " -syncmode=" + CommandExecutor.Quotes + "fullall" + CommandExecutor.Quotes
-                    + " -metadatabinaries=" + CommandExecutor.Quotes + metaBinariesPath + CommandExecutor.Quotes
-                    + " -connect=" + CommandExecutor.Quotes + $"Data Source=localhost;Initial Catalog={scaleUnit.AxDbName};Integrated Security=True;Enlist=True;Application Name=AXVSSDK" + CommandExecutor.Quotes
-                    + " -verbosity=" + CommandExecutor.Quotes + "Diagnostic" + CommandExecutor.Quotes
-                    + " -scaleUnitOptionsAsJson=" + CommandExecutor.Quotes + "{" + CommandExecutor.Quotes + "IsScaleUnitFeatureEnabled" + CommandExecutor.Quotes + ": true, " + CommandExecutor.Quotes + "scaleUnitMnemonics" + CommandExecutor.Quotes + ":" + $"'{scaleUnit.ScaleUnitId}'" + " }" + CommandExecutor.Quotes
-                    + " -triggerOptionsAsJson=" + CommandExecutor.Quotes + "{" + CommandExecutor.Quotes + "IsEnabled" + CommandExecutor.Quotes + ": true}" + CommandExecutor.Quotes
-                    + $" > {scaleUnit.AxDbName}_DbSync.log";
+                string args =
+                    "-syncmode=\"fullall\""
```
[the remark this answers is not in the document]
Comment (Antonstockmarr) [reply src/ScaleUnitManagement/ScaleUnitFeatureManager/Common/RunDBSync.cs:33]: I did test this, and it does work. I believe the reason is that before the operation was called using Process.Start(), with file name set to "powershell". This meant that the string had to be parsed first by the Process object, and then afterwards by PowerShell. So the qoutes needed to be wrapped twice. But now the command is run directly by Process.Start(), with the file name set directly to the dbsSync executable. So the double wrapped quotes are not needed.

File: src/ScaleUnitManagement/ScaleUnitFeatureManager/Utilities/IISAdministrationHelper.cs [hunk 2]
```diff
@@ -133,7 +133,7 @@ private static void CloneScaleUnitWebRoot(ScaleUnitInstance scaleUnit)
                 $@"robocopy {Config.HubScaleUnit().SiteRoot()} {scaleUnit.SiteRoot()} /MIR /MT /log+:{robocopyLogPath};";
 
             CommandExecutor ce = new CommandExecutor(cmd);
-            ce.RunCommand(new List<int>() { 0, 1 });
+            ce.RunCommand(new int[] { 0, 1 });
```
Comment (AndreasHassing) [inline src/ScaleUnitManagement/ScaleUnitFeatureManager/Utilities/IISAdministrationHelper.cs:136]: The cool thing about `params` is that you can now use `RunCommand` like:

```csharp
ce.RunCommand(); // assumes an empty array
ce.RunCommand(1); // passes in a: `new int[] { 1 }`
ce.RunCommand(0, 1); // passes in a: `new int[] { 0, 1 }`
```

**Review by AndreasHassing — CHANGES_REQUESTED** [review]
Fix use of `params` and we're golden. Thanks for this Anton! 👍

File: src/ScaleUnitManagement/ScaleUnitFeatureManager/Utilities/IISAdministrationHelper.cs (hunk 2 above)
Comment (Antonstockmarr) [reply src/ScaleUnitManagement/ScaleUnitFeatureManager/Utilities/IISAdministrationHelper.cs:136]: Ahh, okay that's really cool!

**Review by AndreasHassing — APPROVED** [review]

**Pull request merged**